components/debug/win32.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
We tend to use define-based include guards instead of `#pragma once`. It seems to be ubiquitous these days, so maybe we should reconsider it, but where not absolutely impossible, we stick to standard C++.

@zinnschlag Any opinions?

Reply:
@AnyOldName3 I'd advise to give a free hand for use of either include guard style. Of course I'm abiding by either decision.

The only way `#pragma once` fails is when the same header is included multiple times via hard links, etc. It's not treated to a warning by GNU or Clang either.